shakenfist/baseobject.py

```diff
@@ -58,9 +58,12 @@ def add_event(self, operation, phase, duration=None, msg=None):
     def _db_create(cls, object_uuid, metadata):
         metadata['uuid'] = object_uuid
         etcd.create(cls.object_type, None, object_uuid, metadata)
-        LOG.with_fields({
-            'object_type': cls.object_type
-        }).with_fields(metadata).debug('Object created')
+
+        # Mangle things a little so it matches other log messages for this object
+        log_metadata = metadata.copy()
+        log_metadata[cls.object_type] = log_metadata['uuid']
+        del log_metadata['uuid']
+        LOG.with_fields(metadata).debug('Object created')
```

Comment:
Could I suggest replacing 5 lines with 1 line before the `uuid` is added:
```
LOG.with_fields(metadata).with_field('uuid', object_uuid).debug('Creating object')
```

Reply:
But we want to remove ("uuid", uuid) and replace it with ("instance", uuid). Right?

Reply:
Whoops. Yep.
```
LOG.with_fields(metadata).with_instance(object_uuid).debug('Creating object')
```

Reply:
But that doesn't rename the uuid field to instance? Or am I being confused here?

Reply:
No I am being confused. I forgot this was a class method. So no log method defined.
```
LOG.with_fields(metadata).with_instance(object_uuid).debug('Creating object')
```

Reply:
Oh, the above should read:

```
LOG.with_fields(log_metadata).debug('Object created')
```

Does that clear up the confusion?

Reply:
Thanks for the call. Let us never speak of these things again.